src/sas/qtgui/Plotting/Slicers/Arc.py

```diff
@@ -125,7 +126,8 @@ def move(self, x, y, ev):
         self._mouse_x = x
         self._mouse_y = y
         self.has_move = True
-        self.base.base.update()
+        self.base.update()
+        self.base.draw()
```

Comment:
It would simplify things for sure, but I spent a fair amount of time thinking about ways to simplify before doing this and am pretty sure I contemplated this solution?.. but cannot remember right now why I might have rejected it ☹️. If I cannot remember it soon, probably it is not one I contemplated and thus an obvious improvement.